products/public-websites/README.md

```diff
@@ -777,7 +778,7 @@ Still owned by PW:
 * https://www.va.gov/disability/how-to-file-claim
 * https://www.va.gov/decision-reviews/higher-level-review/
 
-TODO: Learn what is the actual widget component of these pages, and PW ownership of that. Have these all been replaced by online forms? 
+TODO: Learn what is the actual widget component of these pages, and PW ownership of that. Have these all been replaced by online forms? TODO? 
```

Comment:
Little bit more context on these URLs. How do we want to fill in this question?

`https://www.va.gov/education/eligibility`
- React component: src/applications/edu-benefits/component/createEducationApplicationStatus.jsx

`https://www.va.gov/education/how-to-apply`
- React component: src/applications/edu-benefits/component/createEducationApplicationStatus.jsx

`https://www.va.gov/disability/eligibility`
- No React component on this page

`https://www.va.gov/disability/how-to-file-claim`
- No React component on this page

`https://www.va.gov/decision-reviews/higher-level-review`
- No React component on this page

Reply:
Should we just add that info into the page? Seems useful.